lib/Ohmbrewer_Temperature.h

```diff
@@ -8,6 +8,7 @@
 
 #include "application.h"
 
+
```

Comment:
If you drop this space this file won't get touched in the commit... just sayin'...